application/controllers/HostController.php

```diff
@@ -2,21 +2,54 @@
 
 namespace Icinga\Module\Businessprocess\Controllers;
 
-use Icinga\Module\Monitoring\Controller;
+use Icinga\Module\Businessprocess\Common\IcingadbDatabase;
+use Icinga\Module\Businessprocess\Web\Controller;
+use Icinga\Module\Icingadb\Model\Host;
+use Icinga\Module\Monitoring\Backend;
 use Icinga\Web\Url;
 
 class HostController extends Controller
 {
+    use IcingadbDatabase;
+
+    protected $backend;
+
+    protected $allParams;
```

Comment:
You don't need that. Accessing params should be done only with `$this->params->getRequired($name)` and `$this->params->get($name[, $default])`.

Reply:
I have made the changes as discussed in the call.